### PR TITLE
Add Jobs search label to profile node

### PR DESCRIPTION
**Pull request by kristinochka**
Signed-off-by: KRISTINA MAYO <ktopchi@us.ibm.com>

## Proposed changes

https://github.com/zowe/vscode-extension-for-zowe/issues/2062

## Release Notes

Milestone:

Changelog:

## Types of changes

What types of changes does your code introduce to Zowe Explorer?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Updates to Documentation or Tests (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_

- [x] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance) wiki
- [x] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
- [x] PR Description is included
- [x] gif or screenshot is included if visual changes are made
- [x] `yarn workspace vscode-extension-for-zowe vscode:prepublish` has been executed
- [x] All checks have passed (DCO, Jenkins and Code Coverage)
- [x] I have added unit test and it is passing
- [ ] I have added integration test and it is passing
- [ ] There is coverage for the code that I have added
- [ ] I have tested it manually and there are no regressions found
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any PR dependencies have been merged and published (if appropriate)

## Further comments

image

### CODE REVIEW

File: packages/zowe-explorer/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 
 ## TBD Release
 
+- Added Job search label to the Jobs tree.
```
Comment (zFernand0) [inline packages/zowe-explorer/CHANGELOG.md:7]: Should this be under the new enhancements section?
Or maybe we just update the related entry to also mention the label updates?

**Review by zFernand0 — COMMENTED** [review]
The enhancement seems very straightforward and useful.
I left one comment about the changelog.

Q: should we consider making this a bit more dynamic for V3?
For instance, other people may find more value in seeing specific information (that I'm sure we have available) in certain labels.
Just something to think about :yum:

**Comment by kristinochka** [comment]
> Q: should we consider making this a bit more dynamic for V3? For instance, other people may find more value in seeing specific information (that I'm sure we have available) in certain labels.

An example would be helpful here. We can definitely discuss this use case.

File: packages/zowe-explorer/CHANGELOG.md (hunk 1 above)
Comment (kristinochka) [reply packages/zowe-explorer/CHANGELOG.md:7]: I moved this to new enhancements section

**Comment by kristinochka** [comment]
@zFernand0 Ready to review now.

**Comment by zFernand0** [comment]
> `@zFernand0` Ready to review now.

Happy to review it again.
Please keep in mind that this is an enhancement and thus should stay with `main` as the target branch for a future `2.7.0` release of Zowe Explorer.

**Review by zFernand0 — APPROVED** [review]
@kristinochka 
Seems to work as expected.
There seems to be a very annoying issue to work around.
- See #2112

**Comment by kristinochka** [comment]
@zFernand0 Thanks, Billie Jean is out this week, Could we add somebody else to review or merge?

**Comment by kristinochka** [comment]
https://github.com/zowe/vscode-extension-for-zowe/issues/2112 Seems to be a new regression.

**Review by traeok — APPROVED** [review]
LGTM! Thanks @kristinochka 

I also noticed #2112 and was able to make a fix on a local branch - if desired I can make a PR for this 😅

**Comment by zFernand0** [comment]
There might be a slight issue with the updates made to the labels.
Based on @adam-wolfe's comments during yesterday's call, we looked into things a bit closer regarding favorites and found that the updated labels break the searching capabilities from a favorited job-search.
temp

**Pull request merged**